cookpad-performance.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.7.0"
 
   spec.add_dependency "prosopite", "~> 1.0"
-  spec.add_dependency "rails", ">= 7.0"
+  spec.add_dependency "rails", ">= 5.0"
```

Comment:
Is there a reason why this gem has to support Rails 5.0?

Rails 5.0 is already EOL, and 5.2 will stop getting security fix on June 1, 2022.


Allowing this gem to run with the old versions mean that we need to make sure that gems is able to install and run on any supported version of Rails. That probably means adding them to the GitHub workflow test suite matrix to make sure it's being run against that version of Rails.

If you want to be more permissive, I think `>= 6.1.0` will be permissive enough as 6.1 is still getting supported in terms of a security fix (https://guides.rubyonrails.org/maintenance_policy.html#security-issues)

Reply:
> Is there a reason why this gem has to support Rails 5.0?

Nope. Not 5.0 in particular, I just wanted to cover a safe range of versions so that we didn't have to worry so much about dependency clashes in other environments. 

6.1 works for me